presto-docs/src/main/sphinx/release/release-0.233.rst

```diff
@@ -0,0 +1,79 @@
+* Add support for soft affinity scheduling, which makes the best effort to fetch the same piece of data from the same worker,
+  while allowing fallback to random workers if the preferred workers are too busy to handle additional splits.
+  The feature can be turned on by ``ConnectorSplit#getNodeSelectionStrategy()``.
+* Add soft affinity scheduling to bucketed split. It makes the best effort to fetch the same bucket data from the same worker.
+  When using dynamic group scheduling, if the preferred workers are unavailable to handle the specific bucket splits, it will fallback to random workers.
+* Add hash functions ``fnv1_32``, ``fnv1_64``, ``fnv1a_32``, and ``fnv1a_64``.
+* Add IP address function :func:`ip_subnet_min`, :func:`ip_subnet_max`, :func:`ip_subnet_range`, and :func:`is_subnet_of`.
+* Improve performance of ``StreamingAggregationOperator``.
+
+
+Hive Changes
+____________
+* Add directory listing cache for Hive Connector.
+* Allow server to start when ``hive.s3-file-system-type`` is set to ``HADOOP_DEFAULT``.
+* Add configuration property ``hive.insert-overwrite-immutable-partitions-enabled`` to allow admin set insert overwrite
+  as the default insertion behavior for Hive connector.
+* Add configuration property ``hive.node-selection-strategy`` to choose ``NodeSelectionStrategy``. When ``SOFT_AFFINITY`` is selected,
+  scheduler will make the best effort to request the same worker to fetch the same file.
+* Remove configuration property ``hive.force-local-scheduling``. The same functionality can be achieved by setting
+  ``hive.node-selection-strategy`` to ``HARD_AFFINITY``.
+* Store column names and types for views in the metastore. Views in the Hive connector can now only use types supported by Hive.
+
+
+Verifier Changes
+________________
+* Fix an issue where invalid checksum queries can be generated for certain queries containing columns of ``RowType``.
+* Fix an issue where checksum query would fail for queries containing map columns whose key or value types are arrays or rows.
+* Fix incorrect decision for determinism analysis of queries with top-level ``LIMIT`` clause. (:pr:`14176`).
+* Add checks for keys, values, and cardinality sum when validating a map column.
+* Add support to disable individual failure resolvers (:pr:`14148`).
+* Add support to auto-resolve control checksum query failures with ``COMPILER_ERROR``, instead of skipping the verification.
+* Add support for specifying non-deterministic catalogs by configuration property ``determinism.non-determinism-catalogs``.
+  Queries explicitly referencing tables from those catalogs are treated as non-deterministic.
+* Improve query performance during determinism analysis of queries with top-level ``LIMIT`` clause.
+* Improve correctness check for floating point columns whose mean values of either the control query or the test query is closed to 0.
+
+
+Druid Changes
+_____________
+* Add Druid Connector.
+* Improve query performance with filter pushdown.
```

Comment:
We just added the connector, so it's being introduced with this optimization.  Can't we remove this line then?